.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,21 @@
+name: Unit Test
+
+on:
+    pull_request: 
+        branches: []
```

Comment:
no branch specified
```suggestion
        branches: [main, staging]
```

Reply:
The idea is these unit-tests can run on all branches.
Having specific branches does reduce our github action minutes usage, sooo yeah lets do that!